browser/ui/views/profiles/brave_profile_menu_view.cc

```diff
@@ -33,6 +33,9 @@ void BraveProfileMenuView::BuildAutofillButtons() {}
 // We don't want to show any Chromium sync info.
 void BraveProfileMenuView::BuildSyncInfo() {}
 
+// We don't want any feature buttons like MANAGE_GOOGLE_ACCOUNT
+void BraveProfileMenuView::BuildFeatureButtons() {}
```

Comment:
I believe we want to keep the `IDS_PROFILES_CLOSE_X_WINDOWS_BUTTON` here. At least it's part of the intended design change here https://github.com/brave/brave-browser/issues/8807